catalogs/v4.14/catalog/openshift-cert-manager-operator/bundle-v1.11.1.yaml

```diff
@@ -0,0 +1,92 @@
InJlcXVpcmVkIjpbIm5hbWUiXSwidHlwZSI6Im9iamVjdCJ9LCJ0eXBlIjoiYXJyYXkifSwib3ZlcnJpZGVMYWJlbHMiOnsiYWRkaXRpb25hbFByb3BlcnRpZXMiOnsidHlwZSI6InN0cmluZyJ9LCJ0eXBlIjoib2JqZWN0In19LCJ0eXBlIjoib2JqZWN0In19LCJ0eXBlIjoib2JqZWN0In0sInN0YXR1cyI6eyJkZXNjcmlwdGlvbiI6IkNlcnRNYW5hZ2VyU3RhdHVzIGRlZmluZXMgdGhlIG9ic2VydmVkIHN0YXRlIG9mIENlcnRNYW5hZ2VyIiwicHJvcGVydGllcyI6eyJjb25kaXRpb25zIjp7ImRlc2NyaXB0aW9uIjoiY29uZGl0aW9ucyBpcyBhIGxpc3Qgb2YgY29uZGl0aW9ucyBhbmQgdGhlaXIgc3RhdHVzIiwiaXRlbXMiOnsiZGVzY3JpcHRpb24iOiJPcGVyYXRvckNvbmRpdGlvbiBpcyBqdXN0IHRoZSBzdGFuZGFyZCBjb25kaXRpb24gZmllbGRzLiIsInByb3BlcnRpZXMiOnsibGFzdFRyYW5zaXRpb25UaW1lIjp7ImZvcm1hdCI6ImRhdGUtdGltZSIsInR5cGUiOiJzdHJpbmcifSwibWVzc2FnZSI6eyJ0eXBlIjoic3RyaW5nIn0sInJlYXNvbiI6eyJ0eXBlIjoic3RyaW5nIn0sInN0YXR1cyI6eyJ0eXBlIjoic3RyaW5nIn0sInR5cGUiOnsidHlwZSI6InN0cmluZyJ9fSwidHlwZSI6Im9iamVjdCJ9LCJ0eXBlIjoiYXJyYXkifSwiZ2VuZXJhdGlvbnMiOnsiZGVzY3JpcHRpb24iOiJnZW5lcmF0aW9ucyBhcmUgdXNlZCB0byBkZXRlcm1pbmUgd2hlbiBhbiBpdGVtIG5lZWRzIHRvIGJlIHJlY29uY2lsZWQgb3IgaGFzIGNoYW5nZWQgaW4gYSB3YXkgdGhhdCBuZWVkcyBhIHJlYWN0aW9uLiIsIml0ZW1zIjp7ImRlc2NyaXB0aW9uIjoiR2VuZXJhdGlvblN0YXR1cyBrZWVwcyB0cmFjayBvZiB0aGUgZ2VuZXJhdGlvbiBmb3IgYSBnaXZlbiByZXNvdXJjZSBzbyB0aGF0IGRlY2lzaW9ucyBhYm91dCBmb3JjZWQgdXBkYXRlcyBjYW4gYmUgbWFkZS4iLCJwcm9wZXJ0aWVzIjp7Imdyb3VwIjp7ImRlc2NyaXB0aW9uIjoiZ3JvdXAgaXMgdGhlIGdyb3VwIG9mIHRoZSB0aGluZyB5b3UncmUgdHJhY2tpbmciLCJ0eXBlIjoic3RyaW5nIn0sImhhc2giOnsiZGVzY3JpcHRpb24iOiJoYXNoIGlzIGFuIG9wdGlvbmFsIGZpZWxkIHNldCBmb3IgcmVzb3VyY2VzIHdpdGhvdXQgZ2VuZXJhdGlvbiB0aGF0IGFyZSBjb250ZW50IHNlbnNpdGl2ZSBsaWtlIHNlY3JldHMgYW5kIGNvbmZpZ21hcHMiLCJ0eXBlIjoic3RyaW5nIn0sImxhc3RHZW5lcmF0aW9uIjp7ImRlc2NyaXB0aW9uIjoibGFzdEdlbmVyYXRpb24gaXMgdGhlIGxhc3QgZ2VuZXJhdGlvbiBvZiB0aGUgd29ya2xvYWQgY29udHJvbGxlciBpbnZvbHZlZCIsImZvcm1hdCI6ImludDY0IiwidHlwZSI6ImludGVnZXIifSwibmFtZSI6eyJkZXNjcmlwdGlvbiI6Im5hbWUgaXMgdGhlIG5hbWUgb2YgdGhlIHRoaW5nIHlvdSdyZSB0cmFja2luZyIsInR5cGUiOiJzdHJpbmcifSwibmFtZXNwYWNlIjp7ImRlc2NyaXB0aW9uIjoibmFtZXNwYWNlIGlzIHdoZXJlIHRoZSB0aGluZyB5b3UncmUgdHJhY2tpbmcgaXMiLCJ0eXBlIjoic3RyaW5nIn0sInJlc291cmNlIjp7ImRlc2NyaXB0aW9uIjoicmVzb3VyY2UgaXMgdGhlIHJlc291cmNlIHR5cGUgb2YgdGhlIHRoaW5nIHlvdSdyZSB0cmFja2luZyIsInR5cGUiOiJzdHJpbmcifX0sInR5cGUiOiJvYmplY3QifSwidHlwZSI6ImFycmF5In0sIm9ic2VydmVkR2VuZXJhdGlvbiI6eyJkZXNjcmlwdGlvbiI6Im9ic2VydmVkR2VuZXJhdGlvbiBpcyB0aGUgbGFzdCBnZW5lcmF0aW9uIGNoYW5nZSB5b3UndmUgZGVhbHQgd2l0aCIsImZvcm1hdCI6ImludDY0IiwidHlwZSI6ImludGVnZXIifSwicmVhZHlSZXBsaWNhcyI6eyJkZXNjcmlwdGlvbiI6InJlYWR5UmVwbGljYXMgaW5kaWNhdGVzIGhvdyBtYW55IHJlcGxpY2FzIGFyZSByZWFkeSBhbmQgYXQgdGhlIGRlc2lyZWQgc3RhdGUiLCJmb3JtYXQiOiJpbnQzMiIsInR5cGUiOiJpbnRlZ2VyIn0sInZlcnNpb24iOnsiZGVzY3JpcHRpb24iOiJ2ZXJzaW9uIGlzIHRoZSBsZXZlbCB0aGlzIGF2YWlsYWJpbGl0eSBhcHBsaWVzIHRvIiwidHlwZSI6InN0cmluZyJ9fSwidHlwZSI6Im9iamVjdCJ9fSwicmVxdWlyZWQiOlsic3BlYyJdLCJ0eXBlIjoib2JqZWN0In19LCJzZXJ2ZWQiOnRydWUsInN0b3JhZ2UiOnRydWUsInN1YnJlc291cmNlcyI6eyJzdGF0dXMiOnt9fX1dfSwic3RhdHVzIjp7ImFjY2VwdGVkTmFtZXMiOnsia2luZCI6IiIsInBsdXJhbCI6IiJ9LCJjb25kaXRpb25zIjpudWxsLCJzdG9yZWRWZXJzaW9ucyI6bnVsbH19
```

Comment:
Is there a document to understand more about this field? Not sure of what does the data represent here.

Reply:
It's the objects in encoded form. For versions below 4.17, bundle objects will be in encoded form, similar data in 4.17 bundle is readable form like [here](https://github.com/openshift/cert-manager-operator-release/pull/170/files#diff-f05a36b7eded4e9fa3513d8bdfcb4fe5e2917b080c83ffee8a1536ad8cd51482R45).

Reply:
https://olm.operatorframework.io/docs/reference/file-based-catalogs/#olmbundleobject-deprecated